src/pages/MemberProfileScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import { Divider, Stack, Typography } from '@mui/material';
+
+import MemberPersonalInformation from '@/components/main/MemberPersonalInformation';
+import MemberPublicProfile from '@/components/main/MemberPublicProfile';
+import { useAccountTranslation } from '@/config/i18n';
+
+const MemberProfileScreen = (): JSX.Element | null => {
+  const { t } = useAccountTranslation();
+  return (
+    <Stack spacing={2}>
+      <Typography variant="h4" component="h1">
```

Comment:
I like that you used `component` and `variant` but I think we should go for instead:
```suggestion
      <Typography variant="h2" component="h1">
```